banking/ebics/utils.py

```diff
@@ -332,6 +332,83 @@ def upload_camt_file():
 	process_camt_document(camt_document, bank_account)
 
 
+@frappe.whitelist()
+def upload_mt940_file():
+	frappe.has_permission("Bank Transaction", "create", throw=True)
+
+	file_bytes = frappe.local.uploaded_file
+	bank_account = frappe.form_dict.docname
+
+	register_fintech()
+
+	from fintech.swift import parse_mt940
+
+	mt940_data = file_bytes.decode()
+	statements = parse_mt940(mt940_data)
+
+	for statement in statements:
+		process_mt940_statement(statement, bank_account)
+
+
+def process_mt940_statement(statement, bank_account: str):
```

Comment:
```suggestion
def process_mt940_statement(statement: dict, bank_account: str):
```
This is what I got from https://www.joonis.de/en/fintech/doc/swift/#fintech.swift.parse_mt940.